EIPS/eip-X.md

```diff
@@ -0,0 +1,201 @@
+---
+eip: <to be assigned>
+title: Serenity -- Phase 0
```

Comment:
I'm not sure how the `--` will render in the title. It may make sense to update to

```suggestion
title: 'Serenity: Phase 0'
```
or

```suggestion
title: Serenity Phase 0
```